pkg/util/kubernetes/apiserver/apiserver_nocompile.go

```diff
@@ -25,12 +25,12 @@ type APIClient struct{}
 
 // MetadataMapperBundle maps the podNames to the metadata they are associated with.
 type MetadataMapperBundle struct {
-	PodNameToService map[string][]string
-	m                sync.RWMutex
+	Services map[string]map[string][]string
```

Comment:
Is `map[string]map[string][]string` too confusing of a type?

Reply:
yes, I was about to say - I think we would be better off using a struct